src/Misc/PerformanceViewer/performanceViewerCollector.ts

```diff
@@ -197,6 +200,10 @@ export class PerformanceViewerCollector {
 
             this.datasets.ids.push(strategy.id);
 
+            if (category) {
+                category = category.replace(exportedDataSeparator, "");

```

Comment:
replace will only remove the first instance, unless you are using a regex. there is a replaceAll, or better - use a regex:
```suggestion
                category = category.replace(new RegExp(exportedDataSeparator, "g"), "");
```
if you wonder why I use the RegExp constructor instead of /@/g, it's because you can't use variables between the two `/ `

Reply:
Updated!